client/coral-admin/src/components/ModerationKeysModal.js

```diff
@@ -12,6 +12,7 @@ const shortcuts = [
       'k': 'modqueue.prev_comment',
       'ctrl+f': 'modqueue.toggle_search',
       't': 'modqueue.next_queue',
+      '1...5': 'modqueue.jumptoqueue',
```

Comment:
It would super cool if the number `5` here would correspond to the exact number of queues. If you pass over a `queueCount` property from `Moderation`, and turn this from a static object to a function, it should work.